Google.Api.Generator.Tests/ProtoTest.cs

```diff
@@ -227,6 +229,54 @@ void CleanUp(string path)
         [Fact]
         public void UnitTests() => ProtoTestSingle("UnitTests", ignoreCsProj: true, ignoreSnippets: true);
 
+        [Fact]
+        public void GrpcServiceConfig()
+        {
+            using (var grpcServiceConfig = Invoker.TempFile())
```

Comment:
Is there a reason this can't just be a file elsewhere? (Or a resource that you copy to the file system?)

Lots of the other tests use real files, so it feels like this should be able to as well.

Reply:
Done!